[Automation] java run fails for org.scalactic:scalactic_3:3.3.0-alpha.1

## java run on ubuntu-latest / JDK 25 / mode current-defaults

Failure kind: java run
OS: ubuntu-latest
JDK: 25
Native-image mode: current-defaults
Reproducer: `GVM_TCK_NATIVE_IMAGE_MODE="current-defaults" GVM_TCK_LV="3.3.0-alpha.1" ./gradlew clean javaTest -Pcoordinates="org.scalactic:scalactic_3:3.2.19"`
Runner log: https://github.com/oracle/graalvm-reachability-metadata/actions/runs/26053223885/job/76595939854
Last 300 lines of the log:
```console
====================
Testing library: org.scalactic:scalactic_3:3.2.19
Command: `/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/gradlew nativeTest`
Executing test...
-------
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE
> Task :compileJava NO-SOURCE
> Task :compileScala NO-SOURCE
> Task :tck-test-support:compileJava UP-TO-DATE
> Task :processResources NO-SOURCE
> Task :classes UP-TO-DATE
> Task :compileTestJava NO-SOURCE
> Task :tck-test-support:processResources NO-SOURCE
> Task :tck-test-support:classes UP-TO-DATE
> Task :tck-test-support:jar UP-TO-DATE
> Task :compileTestScala
> Task :jar
> Task :checkTestTimeoutAnnotations
> Task :processTestResources
> Task :testClasses

> Task :test FAILED

ObjectMetaAnonymous1Test > privateValueFallsBackToAccessibleInvocation() FAILED
    org.opentest4j.AssertionFailedError: 
    Expecting value to be true but was false
        at java.base@21.0.11/jdk.internal.reflect.DirectConstructorHandleAccessor.newInstance(DirectConstructorHandleAccessor.java:62)
        at java.base@21.0.11/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:502)
        at app//org_scalactic.scalactic_3.ObjectMetaAnonymous1Test.privateValueFallsBackToAccessibleInvocation(ObjectMetaAnonymous1Test.scala:35)

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.scalactic/scalactic_3/3.2.19/build/reports/problems/problems-report.html

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

4 tests completed, 1 failed

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':test'.
> There were failing tests. See the report at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.scalactic/scalactic_3/3.2.19/build/reports/tests/test/index.html

* Try:
> Run with --scan to generate a Build Scan (Powered by Develocity).

BUILD FAILED in 7s
15 actionable tasks: 5 executed, 10 up-to-date

> Task :tckTest FAILED

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/build/reports/problems/problems-report.html

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':tckTest'.
> Test for org.scalactic:scalactic_3:3.2.19 failed with exit code 1.

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to generate a Build Scan (Powered by Develocity).
> Get more help at https://help.gradle.org.

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD FAILED in 9s
11 actionable tasks: 3 executed, 8 up-to-date
Finished stage 'native-image run' for 3.3.0-alpha.1 at 2026-05-18T18:52:28Z with exit code 1 after 10s
❌ ERROR: 'native-image run' for 3.3.0-alpha.1 failed with exit code 1.
SOFT-FAIL[native-image run][3.3.0-alpha.1][GVM_TCK_NATIVE_IMAGE_MODE="current-defaults" GVM_TCK_LV="3.3.0-alpha.1" ./gradlew clean test -Pcoordinates="org.scalactic:scalactic_3:3.2.19"]
========================================================================================
 org.scalactic:scalactic_3:3.2.19:3.3.0-alpha.1 stage javac compile
========================================================================================
Starting stage 'javac compile' for 3.3.0-alpha.1 at 2026-05-18T18:52:28Z
Command: GVM_TCK_NATIVE_IMAGE_MODE="current-defaults" GVM_TCK_LV="3.3.0-alpha.1" ./gradlew clean compileTestJava -Pcoordinates="org.scalactic:scalactic_3:3.2.19"
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE

> Configure project :
GraalVM Reachability Metadata TCK
---------------------------------

> Task :tckClean
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE
> Task :clean

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.scalactic/scalactic_3/3.2.19/build/reports/problems/problems-report.html

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD SUCCESSFUL in 893ms
9 actionable tasks: 1 executed, 8 up-to-date
Consider enabling configuration cache to speed up this build: https://docs.gradle.org/9.1.0/userguide/configuration_cache_enabling.html

> Task :clean

> Task :compileTestJava
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE
> Task :compileJava NO-SOURCE
> Task :compileScala NO-SOURCE
> Task :tck-test-support:compileJava UP-TO-DATE
> Task :processResources NO-SOURCE
> Task :classes UP-TO-DATE
> Task :compileTestJava NO-SOURCE

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.scalactic/scalactic_3/3.2.19/build/reports/problems/problems-report.html

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD SUCCESSFUL in 852ms
9 actionable tasks: 9 up-to-date
Consider enabling configuration cache to speed up this build: https://docs.gradle.org/9.1.0/userguide/configuration_cache_enabling.html

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/build/reports/problems/problems-report.html

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD SUCCESSFUL in 2s
11 actionable tasks: 3 executed, 8 up-to-date
Finished stage 'javac compile' for 3.3.0-alpha.1 at 2026-05-18T18:52:31Z with exit code 0 after 3s
========================================================================================
 org.scalactic:scalactic_3:3.2.19:3.3.0-alpha.1 stage java run
========================================================================================
Starting stage 'java run' for 3.3.0-alpha.1 at 2026-05-18T18:52:31Z
Command: GVM_TCK_NATIVE_IMAGE_MODE="current-defaults" GVM_TCK_LV="3.3.0-alpha.1" ./gradlew clean javaTest -Pcoordinates="org.scalactic:scalactic_3:3.2.19"
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE

> Configure project :
GraalVM Reachability Metadata TCK
---------------------------------

> Task :tckClean
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE
> Task :clean

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.scalactic/scalactic_3/3.2.19/build/reports/problems/problems-report.html

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD SUCCESSFUL in 800ms
9 actionable tasks: 1 executed, 8 up-to-date
Consider enabling configuration cache to speed up this build: https://docs.gradle.org/9.1.0/userguide/configuration_cache_enabling.html

> Task :clean

> Task :javaTest
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE
> Task :checkTestTimeoutAnnotations
> Task :tck-test-support:compileJava UP-TO-DATE
> Task :tck-test-support:processResources NO-SOURCE
> Task :tck-test-support:classes UP-TO-DATE
> Task :compileJava NO-SOURCE
> Task :compileScala NO-SOURCE
> Task :processResources NO-SOURCE
> Task :tck-test-support:jar UP-TO-DATE
> Task :classes UP-TO-DATE
> Task :compileTestJava NO-SOURCE
> Task :compileTestScala
> Task :processTestResources
> Task :testClasses

> Task :test FAILED

ObjectMetaAnonymous1Test > privateValueFallsBackToAccessibleInvocation() FAILED
    org.opentest4j.AssertionFailedError: 
    Expecting value to be true but was false
        at java.base@21.0.11/jdk.internal.reflect.DirectConstructorHandleAccessor.newInstance(DirectConstructorHandleAccessor.java:62)
        at java.base@21.0.11/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:502)
        at app//org_scalactic.scalactic_3.ObjectMetaAnonymous1Test.privateValueFallsBackToAccessibleInvocation(ObjectMetaAnonymous1Test.scala:35)

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.scalactic/scalactic_3/3.2.19/build/reports/problems/problems-report.html

4 tests completed, 1 failed

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':test'.
> There were failing tests. See the report at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.scalactic/scalactic_3/3.2.19/build/reports/tests/test/index.html

* Try:
> Run with --scan to generate a Build Scan (Powered by Develocity).

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD FAILED in 6s
14 actionable tasks: 4 executed, 10 up-to-date

> Task :javaTest FAILED

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/build/reports/problems/problems-report.html

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':javaTest'.
> Java tests failed

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to generate a Build Scan (Powered by Develocity).
> Get more help at https://help.gradle.org.

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD FAILED in 8s
11 actionable tasks: 3 executed, 8 up-to-date
Finished stage 'java run' for 3.3.0-alpha.1 at 2026-05-18T18:52:40Z with exit code 1 after 9s
❌ ERROR: 'java run' for 3.3.0-alpha.1 failed with exit code 1.
FAILED[java run][3.3.0-alpha.1][GVM_TCK_NATIVE_IMAGE_MODE="current-defaults" GVM_TCK_LV="3.3.0-alpha.1" ./gradlew clean javaTest -Pcoordinates="org.scalactic:scalactic_3:3.2.19"]
```

## java run on ubuntu-latest / JDK latest-ea / mode current-defaults

Failure kind: java run
OS: ubuntu-latest
JDK: latest-ea
Native-image mode: current-defaults
Reproducer: `GVM_TCK_NATIVE_IMAGE_MODE="current-defaults" GVM_TCK_LV="3.3.0-alpha.1" ./gradlew clean javaTest -Pcoordinates="org.scalactic:scalactic_3:3.2.19"`
Runner log: https://github.com/oracle/graalvm-reachability-metadata/actions/runs/26053223885/job/76595939853
Last 300 lines of the log:
```console
====================
Testing library: org.scalactic:scalactic_3:3.2.19
Command: `/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/gradlew nativeTest`
Executing test...
-------
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE
> Task :compileJava NO-SOURCE
> Task :tck-test-support:compileJava UP-TO-DATE
> Task :compileScala NO-SOURCE
> Task :tck-test-support:processResources NO-SOURCE
> Task :tck-test-support:classes UP-TO-DATE
> Task :processResources NO-SOURCE
> Task :classes UP-TO-DATE
> Task :tck-test-support:jar UP-TO-DATE
> Task :jar
> Task :compileTestJava NO-SOURCE
> Task :compileTestScala
> Task :checkTestTimeoutAnnotations
> Task :processTestResources
> Task :testClasses

> Task :test FAILED

ObjectMetaAnonymous1Test > privateValueFallsBackToAccessibleInvocation() FAILED
    org.opentest4j.AssertionFailedError: 
    Expecting value to be true but was false
        at java.base@21.0.11/jdk.internal.reflect.DirectConstructorHandleAccessor.newInstance(DirectConstructorHandleAccessor.java:62)
        at java.base@21.0.11/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:502)

        at app//org_scalactic.scalactic_3.ObjectMetaAnonymous1Test.privateValueFallsBackToAccessibleInvocation(ObjectMetaAnonymous1Test.scala:35)

4 tests completed, 1 failed

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':test'.
> There were failing tests. See the report at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.scalactic/scalactic_3/3.2.19/build/reports/tests/test/index.html

* Try:
> Run with --scan to generate a Build Scan (Powered by Develocity).

BUILD FAILED in 7s
[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.scalactic/scalactic_3/3.2.19/build/reports/problems/problems-report.html

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.
15 actionable tasks: 5 executed, 10 up-to-date

> Task :tckTest FAILED

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/build/reports/problems/problems-report.html

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':tckTest'.
> Test for org.scalactic:scalactic_3:3.2.19 failed with exit code 1.

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to generate a Build Scan (Powered by Develocity).
> Get more help at https://help.gradle.org.

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD FAILED in 9s
11 actionable tasks: 3 executed, 8 up-to-date
Finished stage 'native-image run' for 3.3.0-alpha.1 at 2026-05-18T18:52:31Z with exit code 1 after 10s
❌ ERROR: 'native-image run' for 3.3.0-alpha.1 failed with exit code 1.
SOFT-FAIL[native-image run][3.3.0-alpha.1][GVM_TCK_NATIVE_IMAGE_MODE="current-defaults" GVM_TCK_LV="3.3.0-alpha.1" ./gradlew clean test -Pcoordinates="org.scalactic:scalactic_3:3.2.19"]
========================================================================================
 org.scalactic:scalactic_3:3.2.19:3.3.0-alpha.1 stage javac compile
========================================================================================
Starting stage 'javac compile' for 3.3.0-alpha.1 at 2026-05-18T18:52:31Z
Command: GVM_TCK_NATIVE_IMAGE_MODE="current-defaults" GVM_TCK_LV="3.3.0-alpha.1" ./gradlew clean compileTestJava -Pcoordinates="org.scalactic:scalactic_3:3.2.19"
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE

> Configure project :
GraalVM Reachability Metadata TCK
---------------------------------

> Task :tckClean
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE
> Task :clean

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.scalactic/scalactic_3/3.2.19/build/reports/problems/problems-report.html

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD SUCCESSFUL in 760ms
9 actionable tasks: 1 executed, 8 up-to-date
Consider enabling configuration cache to speed up this build: https://docs.gradle.org/9.1.0/userguide/configuration_cache_enabling.html

> Task :clean

> Task :compileTestJava
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE
> Task :compileJava NO-SOURCE
> Task :compileScala NO-SOURCE
> Task :tck-test-support:compileJava UP-TO-DATE
> Task :processResources NO-SOURCE
> Task :classes UP-TO-DATE
> Task :compileTestJava NO-SOURCE

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.scalactic/scalactic_3/3.2.19/build/reports/problems/problems-report.html

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD SUCCESSFUL in 815ms
9 actionable tasks: 9 up-to-date
Consider enabling configuration cache to speed up this build: https://docs.gradle.org/9.1.0/userguide/configuration_cache_enabling.html

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/build/reports/problems/problems-report.html

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD SUCCESSFUL in 2s
11 actionable tasks: 3 executed, 8 up-to-date
Finished stage 'javac compile' for 3.3.0-alpha.1 at 2026-05-18T18:52:34Z with exit code 0 after 3s
========================================================================================
 org.scalactic:scalactic_3:3.2.19:3.3.0-alpha.1 stage java run
========================================================================================
Starting stage 'java run' for 3.3.0-alpha.1 at 2026-05-18T18:52:34Z
Command: GVM_TCK_NATIVE_IMAGE_MODE="current-defaults" GVM_TCK_LV="3.3.0-alpha.1" ./gradlew clean javaTest -Pcoordinates="org.scalactic:scalactic_3:3.2.19"
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE

> Configure project :
GraalVM Reachability Metadata TCK
---------------------------------

> Task :tckClean
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE
> Task :clean

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.scalactic/scalactic_3/3.2.19/build/reports/problems/problems-report.html

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD SUCCESSFUL in 764ms
9 actionable tasks: 1 executed, 8 up-to-date
Consider enabling configuration cache to speed up this build: https://docs.gradle.org/9.1.0/userguide/configuration_cache_enabling.html

> Task :clean

> Task :javaTest
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE
> Task :checkTestTimeoutAnnotations
> Task :tck-test-support:compileJava UP-TO-DATE
> Task :tck-test-support:processResources NO-SOURCE
> Task :tck-test-support:classes UP-TO-DATE
> Task :compileJava NO-SOURCE
> Task :compileScala NO-SOURCE
> Task :processResources NO-SOURCE
> Task :tck-test-support:jar UP-TO-DATE
> Task :classes UP-TO-DATE
> Task :compileTestJava NO-SOURCE
> Task :compileTestScala
> Task :processTestResources
> Task :testClasses

> Task :test FAILED

ObjectMetaAnonymous1Test > privateValueFallsBackToAccessibleInvocation() FAILED
    org.opentest4j.AssertionFailedError: 
    Expecting value to be true but was false
        at java.base@21.0.11/jdk.internal.reflect.DirectConstructorHandleAccessor.newInstance(DirectConstructorHandleAccessor.java:62)
        at java.base@21.0.11/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:502)
        at app//org_scalactic.scalactic_3.ObjectMetaAnonymous1Test.privateValueFallsBackToAccessibleInvocation(ObjectMetaAnonymous1Test.scala:35)

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.scalactic/scalactic_3/3.2.19/build/reports/problems/problems-report.html

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.
4 tests completed, 1 failed

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':test'.
> There were failing tests. See the report at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.scalactic/scalactic_3/3.2.19/build/reports/tests/test/index.html

* Try:
> Run with --scan to generate a Build Scan (Powered by Develocity).

14 actionable tasks: 4 executed, 10 up-to-date
BUILD FAILED in 6s


> Task :javaTest FAILED

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/build/reports/problems/problems-report.html

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':javaTest'.
> Java tests failed

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to generate a Build Scan (Powered by Develocity).
> Get more help at https://help.gradle.org.

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD FAILED in 8s
11 actionable tasks: 3 executed, 8 up-to-date
Finished stage 'java run' for 3.3.0-alpha.1 at 2026-05-18T18:52:43Z with exit code 1 after 9s
❌ ERROR: 'java run' for 3.3.0-alpha.1 failed with exit code 1.
FAILED[java run][3.3.0-alpha.1][GVM_TCK_NATIVE_IMAGE_MODE="current-defaults" GVM_TCK_LV="3.3.0-alpha.1" ./gradlew clean javaTest -Pcoordinates="org.scalactic:scalactic_3:3.2.19"]
```

## java run on ubuntu-latest / JDK latest-ea / mode future-defaults-all

Failure kind: java run
OS: ubuntu-latest
JDK: latest-ea
Native-image mode: future-defaults-all
Reproducer: `GVM_TCK_NATIVE_IMAGE_MODE="future-defaults-all" GVM_TCK_LV="3.3.0-alpha.1" ./gradlew clean javaTest -Pcoordinates="org.scalactic:scalactic_3:3.2.19"`
Runner log: https://github.com/oracle/graalvm-reachability-metadata/actions/runs/26053223885/job/76595939844
Last 300 lines of the log:
```console
====================
Testing library: org.scalactic:scalactic_3:3.2.19
Command: `/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/gradlew nativeTest`
Executing test...
-------
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE
> Task :compileJava NO-SOURCE
> Task :tck-test-support:compileJava UP-TO-DATE
> Task :compileScala NO-SOURCE
> Task :tck-test-support:processResources NO-SOURCE
> Task :processResources NO-SOURCE
> Task :classes UP-TO-DATE
> Task :tck-test-support:classes UP-TO-DATE
> Task :tck-test-support:jar UP-TO-DATE
> Task :jar
> Task :compileTestJava NO-SOURCE
> Task :compileTestScala
> Task :checkTestTimeoutAnnotations
> Task :processTestResources
> Task :testClasses

> Task :test FAILED

ObjectMetaAnonymous1Test > privateValueFallsBackToAccessibleInvocation() FAILED
    org.opentest4j.AssertionFailedError: 
    Expecting value to be true but was false
        at java.base@21.0.11/jdk.internal.reflect.DirectConstructorHandleAccessor.newInstance(DirectConstructorHandleAccessor.java:62)
        at java.base@21.0.11/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:502)
        at app//org_scalactic.scalactic_3.ObjectMetaAnonymous1Test.privateValueFallsBackToAccessibleInvocation(ObjectMetaAnonymous1Test.scala:35)

4 tests completed, 1 failed

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':test'.
> There were failing tests. See the report at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.scalactic/scalactic_3/3.2.19/build/reports/tests/test/index.html

* Try:
> Run with --scan to generate a Build Scan (Powered by Develocity).


[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.scalactic/scalactic_3/3.2.19/build/reports/problems/problems-report.html

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.
15 actionable tasks: 5 executed, 10 up-to-date
BUILD FAILED in 7s

> Task :tckTest FAILED

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/build/reports/problems/problems-report.html

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':tckTest'.
> Test for org.scalactic:scalactic_3:3.2.19 failed with exit code 1.

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to generate a Build Scan (Powered by Develocity).
> Get more help at https://help.gradle.org.

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD FAILED in 9s
11 actionable tasks: 3 executed, 8 up-to-date
Finished stage 'native-image run' for 3.3.0-alpha.1 at 2026-05-18T18:52:33Z with exit code 1 after 9s
❌ ERROR: 'native-image run' for 3.3.0-alpha.1 failed with exit code 1.
SOFT-FAIL[native-image run][3.3.0-alpha.1][GVM_TCK_NATIVE_IMAGE_MODE="future-defaults-all" GVM_TCK_LV="3.3.0-alpha.1" ./gradlew clean test -Pcoordinates="org.scalactic:scalactic_3:3.2.19"]
========================================================================================
 org.scalactic:scalactic_3:3.2.19:3.3.0-alpha.1 stage javac compile
========================================================================================
Starting stage 'javac compile' for 3.3.0-alpha.1 at 2026-05-18T18:52:33Z
Command: GVM_TCK_NATIVE_IMAGE_MODE="future-defaults-all" GVM_TCK_LV="3.3.0-alpha.1" ./gradlew clean compileTestJava -Pcoordinates="org.scalactic:scalactic_3:3.2.19"
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE

> Configure project :
GraalVM Reachability Metadata TCK
---------------------------------

> Task :tckClean
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE
> Task :clean

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.scalactic/scalactic_3/3.2.19/build/reports/problems/problems-report.html

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD SUCCESSFUL in 842ms
9 actionable tasks: 1 executed, 8 up-to-date
Consider enabling configuration cache to speed up this build: https://docs.gradle.org/9.1.0/userguide/configuration_cache_enabling.html

> Task :clean

> Task :compileTestJava
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE
> Task :compileJava NO-SOURCE
> Task :tck-test-support:compileJava UP-TO-DATE
> Task :compileScala NO-SOURCE
> Task :processResources NO-SOURCE
> Task :classes UP-TO-DATE
> Task :compileTestJava NO-SOURCE

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.scalactic/scalactic_3/3.2.19/build/reports/problems/problems-report.html

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD SUCCESSFUL in 867ms
9 actionable tasks: 9 up-to-date
Consider enabling configuration cache to speed up this build: https://docs.gradle.org/9.1.0/userguide/configuration_cache_enabling.html

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/build/reports/problems/problems-report.html

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD SUCCESSFUL in 2s
11 actionable tasks: 3 executed, 8 up-to-date
Finished stage 'javac compile' for 3.3.0-alpha.1 at 2026-05-18T18:52:36Z with exit code 0 after 3s
========================================================================================
 org.scalactic:scalactic_3:3.2.19:3.3.0-alpha.1 stage java run
========================================================================================
Starting stage 'java run' for 3.3.0-alpha.1 at 2026-05-18T18:52:36Z
Command: GVM_TCK_NATIVE_IMAGE_MODE="future-defaults-all" GVM_TCK_LV="3.3.0-alpha.1" ./gradlew clean javaTest -Pcoordinates="org.scalactic:scalactic_3:3.2.19"
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE

> Configure project :
GraalVM Reachability Metadata TCK
---------------------------------

> Task :tckClean
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE
> Task :clean

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.scalactic/scalactic_3/3.2.19/build/reports/problems/problems-report.html

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD SUCCESSFUL in 851ms
9 actionable tasks: 1 executed, 8 up-to-date
Consider enabling configuration cache to speed up this build: https://docs.gradle.org/9.1.0/userguide/configuration_cache_enabling.html

> Task :clean

> Task :javaTest
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE
> Task :checkTestTimeoutAnnotations
> Task :tck-test-support:compileJava UP-TO-DATE
> Task :tck-test-support:processResources NO-SOURCE
> Task :tck-test-support:classes UP-TO-DATE
> Task :compileJava NO-SOURCE
> Task :compileScala NO-SOURCE
> Task :processResources NO-SOURCE
> Task :classes UP-TO-DATE
> Task :tck-test-support:jar UP-TO-DATE
> Task :compileTestJava NO-SOURCE
> Task :compileTestScala
> Task :processTestResources
> Task :testClasses

> Task :test FAILED

ObjectMetaAnonymous1Test > privateValueFallsBackToAccessibleInvocation() FAILED
    org.opentest4j.AssertionFailedError: 
    Expecting value to be true but was false
        at java.base@21.0.11/jdk.internal.reflect.DirectConstructorHandleAccessor.newInstance(DirectConstructorHandleAccessor.java:62)
        at java.base@21.0.11/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:502)
        at app//org_scalactic.scalactic_3.ObjectMetaAnonymous1Test.privateValueFallsBackToAccessibleInvocation(ObjectMetaAnonymous1Test.scala:35)

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.scalactic/scalactic_3/3.2.19/build/reports/problems/problems-report.html

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.
14 actionable tasks: 4 executed, 10 up-to-date

4 tests completed, 1 failed

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':test'.
> There were failing tests. See the report at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.scalactic/scalactic_3/3.2.19/build/reports/tests/test/index.html

* Try:
> Run with --scan to generate a Build Scan (Powered by Develocity).

BUILD FAILED in 6s

> Task :javaTest FAILED

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/build/reports/problems/problems-report.html

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':javaTest'.
> Java tests failed

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to generate a Build Scan (Powered by Develocity).
> Get more help at https://help.gradle.org.

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD FAILED in 8s
11 actionable tasks: 3 executed, 8 up-to-date
Finished stage 'java run' for 3.3.0-alpha.1 at 2026-05-18T18:52:45Z with exit code 1 after 9s
❌ ERROR: 'java run' for 3.3.0-alpha.1 failed with exit code 1.
FAILED[java run][3.3.0-alpha.1][GVM_TCK_NATIVE_IMAGE_MODE="future-defaults-all" GVM_TCK_LV="3.3.0-alpha.1" ./gradlew clean javaTest -Pcoordinates="org.scalactic:scalactic_3:3.2.19"]
```

--- a/metadata/org.scalactic/scalactic_3/3.3.0-alpha.1/reachability-metadata.json
+++ b/metadata/org.scalactic/scalactic_3/3.3.0-alpha.1/reachability-metadata.json
@@ -1,66 +1,42 @@
 {
-  "reflection": [
+  "reflection" : [
     {
-      "condition": {
-        "typeReached": "org.scalactic.Resources$"
+      "condition" : {
+        "typeReached" : "org.scalactic.Resources$"
       },
-      "type": "java.lang.StackTraceElement[]"
+      "type" : "java.lang.StackTraceElement[]"
     },
     {
-      "condition": {
-        "typeReached": "org.scalactic.source.ObjectMeta$$anon$1"
+      "condition" : {
+        "typeReached" : "org.scalactic.source.ObjectMeta$$anon$1"
       },
-      "type": "java.lang.String[]"
+      "type" : "java.lang.String[]"
     },
     {
-      "condition": {
-        "typeReached": "org.scalactic.source.ObjectMeta$$anon$1"
+      "condition" : {
+        "typeReached" : "org.scalactic.source.ObjectMeta$$anon$1"
       },
-      "type": "java.lang.reflect.Constructor[]"
+      "type" : "java.lang.reflect.Constructor[]"
     },
     {
-      "condition": {
-        "typeReached": "org.scalactic.source.ObjectMeta$$anon$1"
+      "condition" : {
+        "typeReached" : "org.scalactic.source.ObjectMeta$$anon$1"
       },
-      "type": "java.lang.reflect.Field[]"
+      "type" : "java.lang.reflect.Field[]"
     },
     {
-      "condition": {
-        "typeReached": "org.scalactic.source.ObjectMeta$$anon$1"
+      "condition" : {
+        "typeReached" : "org.scalactic.source.ObjectMeta$$anon$1"
       },
-      "type": "java.lang.reflect.Method[]"
-    },
-    {
-      "condition": {
-        "typeReached": "org.scalactic.source.ObjectMeta$$anon$1"
-      },
-      "type": "org_scalactic.scalactic_3.ObjectMetaPrivateFixture"
-    },
-    {
-      "condition": {
-        "typeReached": "org.scalactic.source.ObjectMeta$$anon$1"
-      },
-      "type": "org_scalactic.scalactic_3.ObjectMetaSpecializedAccessorFixture"
-    },
-    {
-      "condition": {
-        "typeReached": "org.scalactic.source.ObjectMeta$$anon$1$$anon$2"
-      },
-      "type": "org_scalactic.scalactic_3.ObjectMetaSpecializedAccessorFixture"
-    },
-    {
-      "condition": {
-        "typeReached": "org.scalactic.source.ObjectMeta$$anon$1"
-      },
-      "type": "org_scalactic.scalactic_3.ObjectMetaVisibleFixture"
+      "type" : "java.lang.reflect.Method[]"
     }
   ],
-  "resources": [
+  "resources" : [
     {
-      "condition": {
-        "typeReached": "org.scalactic.Resources$"
+      "condition" : {
+        "typeReached" : "org.scalactic.Resources$"
       },
-      "bundle": "org.scalactic.ScalacticBundle"
+      "bundle" : "org.scalactic.ScalacticBundle"
     }
   ]
 }

--- a/metadata/org.scalactic/scalactic_3/3.3.0-alpha.1/reachability-metadata.json
+++ b/metadata/org.scalactic/scalactic_3/3.3.0-alpha.1/reachability-metadata.json
@@ -1,0 +1,66 @@
+{
+  "reflection": [
+    {
+      "condition": {
+        "typeReached": "org.scalactic.Resources$"
+      },
+      "type": "java.lang.StackTraceElement[]"
+    },
+    {
+      "condition": {
+        "typeReached": "org.scalactic.source.ObjectMeta$$anon$1"
+      },
+      "type": "java.lang.String[]"
+    },
+    {
+      "condition": {
+        "typeReached": "org.scalactic.source.ObjectMeta$$anon$1"
+      },
+      "type": "java.lang.reflect.Constructor[]"
+    },
+    {
+      "condition": {
+        "typeReached": "org.scalactic.source.ObjectMeta$$anon$1"
+      },
+      "type": "java.lang.reflect.Field[]"
+    },
+    {
+      "condition": {
+        "typeReached": "org.scalactic.source.ObjectMeta$$anon$1"
+      },
+      "type": "java.lang.reflect.Method[]"
+    },
+    {
+      "condition": {
+        "typeReached": "org.scalactic.source.ObjectMeta$$anon$1"
+      },
+      "type": "org_scalactic.scalactic_3.ObjectMetaPrivateFixture"
+    },
+    {
+      "condition": {
+        "typeReached": "org.scalactic.source.ObjectMeta$$anon$1"
+      },
+      "type": "org_scalactic.scalactic_3.ObjectMetaSpecializedAccessorFixture"
+    },
+    {
+      "condition": {
+        "typeReached": "org.scalactic.source.ObjectMeta$$anon$1$$anon$2"
+      },
+      "type": "org_scalactic.scalactic_3.ObjectMetaSpecializedAccessorFixture"
+    },
+    {
+      "condition": {
+        "typeReached": "org.scalactic.source.ObjectMeta$$anon$1"
+      },
+      "type": "org_scalactic.scalactic_3.ObjectMetaVisibleFixture"
+    }
+  ],
+  "resources": [
+    {
+      "condition": {
+        "typeReached": "org.scalactic.Resources$"
+      },
+      "bundle": "org.scalactic.ScalacticBundle"
+    }
+  ]
+}

--- a/metadata/org.scalactic/scalactic_3/index.json
+++ b/metadata/org.scalactic/scalactic_3/index.json
@@ -1,6 +1,19 @@
 [
   {
     "latest" : true,
+    "metadata-version" : "3.3.0-alpha.1",
+    "language" : {
+      "name" : "scala",
+      "version" : "3"
+    },
+    "tested-versions" : [
+      "3.3.0-alpha.1"
+    ],
+    "allowed-packages" : [
+      "org.scalactic"
+    ]
+  },
+  {
     "metadata-version" : "3.2.19",
     "source-code-url" : "https://repo.maven.apache.org/maven2/org/scalactic/scalactic_3/$version$/scalactic_3-$version$-sources.jar",
     "repository-url" : "https://github.com/scalatest/scalatest",

--- a/metadata/org.scalactic/scalactic_3/index.json
+++ b/metadata/org.scalactic/scalactic_3/index.json
@@ -2,6 +2,11 @@
   {
     "latest" : true,
     "metadata-version" : "3.3.0-alpha.1",
+    "source-code-url" : "https://repo.maven.apache.org/maven2/org/scalactic/scalactic_3/$version$/scalactic_3-$version$-sources.jar",
+    "repository-url" : "https://github.com/scalatest/scalatest",
+    "test-code-url" : "https://github.com/scalatest/scalatest/tree/feature-$version$/jvm/scalactic-test/src/test",
+    "documentation-url" : "https://repo.maven.apache.org/maven2/org/scalactic/scalactic_3/$version$/scalactic_3-$version$-javadoc.jar",
+    "description" : "Scalactic provides core constructs used by ScalaTest for expressive assertions and validation, including equality, normalization, value constraints, and Or/Every data types. The scalactic_3 artifact is the Scala 3 JVM build and can be used independently by Scala applications that want those testing and validation utilities without depending on the full ScalaTest suite.",
     "language" : {
       "name" : "scala",
       "version" : "3"

--- a/stats/org.scalactic/scalactic_3/3.3.0-alpha.1/execution-metrics.json
+++ b/stats/org.scalactic/scalactic_3/3.3.0-alpha.1/execution-metrics.json
@@ -1,0 +1,111 @@
+{
+  "fix_java_run_fail:2026-05-18": {
+    "timestamp": "2026-05-18T20:40:25.359582Z",
+    "library": "org.scalactic:scalactic_3:3.3.0-alpha.1",
+    "starting_commit": "d32e7da88422484c88610468f5e66862bf162eb3",
+    "ending_commit": "462c1628ab46fc391bef7e241b8472c940dbdffc",
+    "previous_library": "org.scalactic:scalactic_3:3.2.19",
+    "strategy_name": "java_run_iterative_with_coverage_sources_pi_gpt-5.5",
+    "agent": "pi",
+    "model": "oca/gpt-5.5",
+    "status": "success",
+    "stats": {
+      "version": "3.3.0-alpha.1",
+      "dynamicAccess": {
+        "breakdown": {
+          "reflection": {
+            "coverageRatio": 1.0,
+            "coveredCalls": 7,
+            "totalCalls": 7
+          },
+          "resources": {
+            "coverageRatio": 1.0,
+            "coveredCalls": 1,
+            "totalCalls": 1
+          }
+        },
+        "coverageRatio": 1.0,
+        "coveredCalls": 8,
+        "totalCalls": 8
+      },
+      "libraryCoverage": {
+        "instruction": {
+          "covered": 543,
+          "missed": 114602,
+          "ratio": 0.004716,
+          "total": 115145
+        },
+        "line": {
+          "covered": 59,
+          "missed": 7653,
+          "ratio": 0.00765,
+          "total": 7712
+        },
+        "method": {
+          "covered": 44,
+          "missed": 16233,
+          "ratio": 0.002703,
+          "total": 16277
+        }
+      }
+    },
+    "previous_library_stats": {
+      "version": "3.2.19",
+      "dynamicAccess": {
+        "breakdown": {
+          "reflection": {
+            "coverageRatio": 1.0,
+            "coveredCalls": 6,
+            "totalCalls": 6
+          },
+          "resources": {
+            "coverageRatio": 1.0,
+            "coveredCalls": 1,
+            "totalCalls": 1
+          }
+        },
+        "coverageRatio": 1.0,
+        "coveredCalls": 7,
+        "totalCalls": 7
+      },
+      "libraryCoverage": {
+        "instruction": {
+          "covered": 481,
+          "missed": 115667,
+          "ratio": 0.004141,
+          "total": 116148
+        },
+        "line": {
+          "covered": 61,
+          "missed": 7658,
+          "ratio": 0.007903,
+          "total": 7719
+        },
+        "method": {
+          "covered": 39,
+          "missed": 16228,
+          "ratio": 0.002397,
+          "total": 16267
+        }
+      }
+    },
+    "metrics": {
+      "input_tokens_used": 20761,
+      "cached_input_tokens_used": 173568,
+      "output_tokens_used": 2538,
+      "iterations": 1,
+      "cost_usd": 0.1629,
+      "tested_library_loc": 59,
+      "metadata_entries": 6,
+      "test_only_metadata_entries": 11,
+      "previous_library_metadata_entries": 11,
+      "previous_library_test_only_metadata_entries": 8,
+      "code_coverage_percent": 0.77,
+      "previous_library_coverage_percent": 0.79
+    },
+    "artifacts": {
+      "test_file": "tests/src/org.scalactic/scalactic_3/3.3.0-alpha.1/src/test/scala/org_scalactic/scalactic_3/ObjectMetaAnonymous1Anonymous2Test.scala",
+      "metadata_file": "metadata/org.scalactic/scalactic_3/3.3.0-alpha.1/reachability-metadata.json"
+    }
+  }
+}

--- a/stats/org.scalactic/scalactic_3/3.3.0-alpha.1/stats.json
+++ b/stats/org.scalactic/scalactic_3/3.3.0-alpha.1/stats.json
@@ -1,0 +1,42 @@
+{
+  "versions" : [ {
+    "version" : "3.3.0-alpha.1",
+    "dynamicAccess" : {
+      "breakdown" : {
+        "reflection" : {
+          "coverageRatio" : 1.0,
+          "coveredCalls" : 7,
+          "totalCalls" : 7
+        },
+        "resources" : {
+          "coverageRatio" : 1.0,
+          "coveredCalls" : 1,
+          "totalCalls" : 1
+        }
+      },
+      "coverageRatio" : 1.0,
+      "coveredCalls" : 8,
+      "totalCalls" : 8
+    },
+    "libraryCoverage" : {
+      "instruction" : {
+        "covered" : 543,
+        "missed" : 114602,
+        "ratio" : 0.004716,
+        "total" : 115145
+      },
+      "line" : {
+        "covered" : 59,
+        "missed" : 7653,
+        "ratio" : 0.00765,
+        "total" : 7712
+      },
+      "method" : {
+        "covered" : 44,
+        "missed" : 16233,
+        "ratio" : 0.002703,
+        "total" : 16277
+      }
+    }
+  } ]
+}

--- a/tests/src/org.scalactic/scalactic_3/3.3.0-alpha.1/.gitignore
+++ b/tests/src/org.scalactic/scalactic_3/3.3.0-alpha.1/.gitignore
@@ -1,0 +1,4 @@
+gradlew.bat
+gradlew
+gradle/
+build/

--- a/tests/src/org.scalactic/scalactic_3/3.3.0-alpha.1/build.gradle
+++ b/tests/src/org.scalactic/scalactic_3/3.3.0-alpha.1/build.gradle
@@ -1,0 +1,36 @@
+/*
+ * Copyright and related rights waived via CC0
+ *
+ * You should have received a copy of the CC0 legalcode along with this
+ * work. If not, see <http://creativecommons.org/publicdomain/zero/1.0/>.
+ */
+plugins {
+    id "org.graalvm.internal.tck"
+    id "scala"
+}
+
+String libraryVersion = tck.testedLibraryVersion.get()
+
+dependencies {
+    testImplementation "org.scalactic:scalactic_3:$libraryVersion"
+    testImplementation 'org.assertj:assertj-core:3.22.0'
+    testImplementation 'org.scala-lang:scala3-library_3:3.3.6'
+    testImplementation 'org.scala-lang:scala3-compiler_3:3.3.6'
+}
+
+java {
+    toolchain {
+        languageVersion = JavaLanguageVersion.of(21)
+    }
+}
+
+graalvmNative {
+    agent {
+        defaultMode = "conditional"
+        modes {
+            conditional {
+                userCodeFilterPath = "user-code-filter.json"
+            }
+        }
+    }
+}

--- a/tests/src/org.scalactic/scalactic_3/3.3.0-alpha.1/gradle.properties
+++ b/tests/src/org.scalactic/scalactic_3/3.3.0-alpha.1/gradle.properties
@@ -1,0 +1,6 @@
+# Optional explicit tested library coordinates (format: group:artifact:version).
+# If omitted, fallback resolution in org.graalvm.internal.tck.gradle still uses
+# environment variable GVM_TCK_LC and then this property.
+library.coordinates = org.scalactic:scalactic_3:3.3.0-alpha.1
+library.version = 3.3.0-alpha.1
+metadata.dir = org.scalactic/scalactic_3/3.3.0-alpha.1/

--- a/tests/src/org.scalactic/scalactic_3/3.3.0-alpha.1/settings.gradle
+++ b/tests/src/org.scalactic/scalactic_3/3.3.0-alpha.1/settings.gradle
@@ -1,0 +1,20 @@
+/*
+ * Copyright and related rights waived via CC0
+ *
+ * You should have received a copy of the CC0 legalcode along with this
+ * work. If not, see <http://creativecommons.org/publicdomain/zero/1.0/>.
+ */
+pluginManagement {
+    def tckPath = Objects.requireNonNullElse(
+            System.getenv("GVM_TCK_TCKDIR"),
+            "../../../../tck-build-logic"
+    )
+    includeBuild(tckPath)
+    repositories { mavenLocal(); gradlePluginPortal() }
+}
+
+plugins {
+    id "org.graalvm.internal.tck-settings" version "1.0.0-SNAPSHOT"
+}
+
+rootProject.name = 'org.scalactic.scalactic_3_tests'

--- a/tests/src/org.scalactic/scalactic_3/3.3.0-alpha.1/src/test/resources/META-INF/native-image/reachability-metadata.json
+++ b/tests/src/org.scalactic/scalactic_3/3.3.0-alpha.1/src/test/resources/META-INF/native-image/reachability-metadata.json
@@ -1,0 +1,50 @@
+{
+  "reflection" : [
+    {
+      "condition" : {
+        "typeReached" : "org.scalactic.source.ObjectMeta$$anon$1"
+      },
+      "type" : "org_scalactic.scalactic_3.ObjectMetaPrivateFixture",
+      "methods" : [
+        {
+          "name" : "secret",
+          "parameterTypes" : [ ]
+        }
+      ]
+    },
+    {
+      "condition" : {
+        "typeReached" : "org.scalactic.source.ObjectMeta$$anon$1"
+      },
+      "type" : "org_scalactic.scalactic_3.ObjectMetaSpecializedAccessorFixture",
+      "methods" : [
+        {
+          "name" : "score$mcI$sp",
+          "parameterTypes" : [ ]
+        }
+      ]
+    },
+    {
+      "condition" : {
+        "typeReached" : "org.scalactic.source.ObjectMeta$$anon$1$$anon$2"
+      },
+      "type" : "org_scalactic.scalactic_3.ObjectMetaSpecializedAccessorFixture"
+    },
+    {
+      "condition" : {
+        "typeReached" : "org.scalactic.source.ObjectMeta$$anon$1"
+      },
+      "type" : "org_scalactic.scalactic_3.ObjectMetaVisibleFixture",
+      "methods" : [
+        {
+          "name" : "count",
+          "parameterTypes" : [ ]
+        },
+        {
+          "name" : "name",
+          "parameterTypes" : [ ]
+        }
+      ]
+    }
+  ]
+}

--- a/tests/src/org.scalactic/scalactic_3/3.3.0-alpha.1/src/test/resources/META-INF/native-image/reachability-metadata.json
+++ b/tests/src/org.scalactic/scalactic_3/3.3.0-alpha.1/src/test/resources/META-INF/native-image/reachability-metadata.json
@@ -45,6 +45,24 @@
           "parameterTypes" : [ ]
         }
       ]
+    },
+    {
+      "condition" : {
+        "typeReached" : "org.scalactic.source.ObjectMeta$$anon$1"
+      },
+      "type" : "org_scalactic.scalactic_3.ObjectMetaPrivateFixture"
+    },
+    {
+      "condition" : {
+        "typeReached" : "org.scalactic.source.ObjectMeta$$anon$1"
+      },
+      "type" : "org_scalactic.scalactic_3.ObjectMetaSpecializedAccessorFixture"
+    },
+    {
+      "condition" : {
+        "typeReached" : "org.scalactic.source.ObjectMeta$$anon$1"
+      },
+      "type" : "org_scalactic.scalactic_3.ObjectMetaVisibleFixture"
     }
   ]
 }

--- a/tests/src/org.scalactic/scalactic_3/3.3.0-alpha.1/src/test/scala/org_scalactic/scalactic_3/ObjectMetaAnonymous1Anonymous2Test.scala
+++ b/tests/src/org.scalactic/scalactic_3/3.3.0-alpha.1/src/test/scala/org_scalactic/scalactic_3/ObjectMetaAnonymous1Anonymous2Test.scala
@@ -1,0 +1,30 @@
+/*
+ * Copyright and related rights waived via CC0
+ *
+ * You should have received a copy of the CC0 legalcode along with this
+ * work. If not, see <http://creativecommons.org/publicdomain/zero/1.0/>.
+ */
+package org_scalactic.scalactic_3
+
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+import org.scalactic.source.ObjectMeta
+
+import scala.annotation.targetName
+
+class ObjectMetaAnonymous1Anonymous2Test {
+  @Test
+  def valueFallsBackToSpecializedAccessorName(): Unit = {
+    val fixture: ObjectMetaSpecializedAccessorFixture = new ObjectMetaSpecializedAccessorFixture(42)
+    val meta: ObjectMeta = ObjectMeta(fixture)
+
+    assertThat(meta.value("score")).isEqualTo(42)
+  }
+}
+
+final class ObjectMetaSpecializedAccessorFixture(scoreInput: Int) {
+  private[this] val storedScore: Int = scoreInput
+
+  @targetName("score$mcI$sp")
+  private def specializedScore: Int = storedScore
+}

--- a/tests/src/org.scalactic/scalactic_3/3.3.0-alpha.1/src/test/scala/org_scalactic/scalactic_3/ObjectMetaAnonymous1Test.scala
+++ b/tests/src/org.scalactic/scalactic_3/3.3.0-alpha.1/src/test/scala/org_scalactic/scalactic_3/ObjectMetaAnonymous1Test.scala
@@ -10,8 +10,6 @@ import org.assertj.core.api.Assertions.assertThat
 import org.junit.jupiter.api.Test
 import org.scalactic.source.ObjectMeta
 
-import scala.annotation.targetName
-
 class ObjectMetaAnonymous1Test {
   @Test
   def fieldNamesAndPublicValuesUseJavaReflection(): Unit = {
@@ -39,9 +37,4 @@ class ObjectMetaAnonymous1Test {
 
 final case class ObjectMetaVisibleFixture(name: String, count: Int)
 
-final class ObjectMetaPrivateFixture(secretInput: String) {
-  private[this] val secret: String = secretInput
-
-  @targetName("secret")
-  private def hiddenSecret: String = secret
-}
+final class ObjectMetaPrivateFixture(private val secret: String)

--- a/tests/src/org.scalactic/scalactic_3/3.3.0-alpha.1/src/test/scala/org_scalactic/scalactic_3/ObjectMetaAnonymous1Test.scala
+++ b/tests/src/org.scalactic/scalactic_3/3.3.0-alpha.1/src/test/scala/org_scalactic/scalactic_3/ObjectMetaAnonymous1Test.scala
@@ -1,0 +1,47 @@
+/*
+ * Copyright and related rights waived via CC0
+ *
+ * You should have received a copy of the CC0 legalcode along with this
+ * work. If not, see <http://creativecommons.org/publicdomain/zero/1.0/>.
+ */
+package org_scalactic.scalactic_3
+
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+import org.scalactic.source.ObjectMeta
+
+import scala.annotation.targetName
+
+class ObjectMetaAnonymous1Test {
+  @Test
+  def fieldNamesAndPublicValuesUseJavaReflection(): Unit = {
+    val fixture: ObjectMetaVisibleFixture = ObjectMetaVisibleFixture("Ada", 37)
+    val meta: ObjectMeta = ObjectMeta(fixture)
+
+    assertThat(meta.fieldNames.contains("name")).isTrue()
+    assertThat(meta.fieldNames.contains("count")).isTrue()
+    assertThat(meta.hasField("name")).isTrue()
+    assertThat(meta.value("name")).isEqualTo("Ada")
+    assertThat(meta.value("count")).isEqualTo(37)
+    assertThat(meta.typeName("name")).isEqualTo(classOf[String].getName)
+    assertThat(meta.shortTypeName("count")).isEqualTo("Integer")
+  }
+
+  @Test
+  def privateValueFallsBackToAccessibleInvocation(): Unit = {
+    val fixture: ObjectMetaPrivateFixture = new ObjectMetaPrivateFixture("classified")
+    val meta: ObjectMeta = ObjectMeta(fixture)
+
+    assertThat(meta.fieldNames.contains("secret")).isTrue()
+    assertThat(meta.value("secret")).isEqualTo("classified")
+  }
+}
+
+final case class ObjectMetaVisibleFixture(name: String, count: Int)
+
+final class ObjectMetaPrivateFixture(secretInput: String) {
+  private[this] val secret: String = secretInput
+
+  @targetName("secret")
+  private def hiddenSecret: String = secret
+}

--- a/tests/src/org.scalactic/scalactic_3/3.3.0-alpha.1/src/test/scala/org_scalactic/scalactic_3/ResourcesTest.scala
+++ b/tests/src/org.scalactic/scalactic_3/3.3.0-alpha.1/src/test/scala/org_scalactic/scalactic_3/ResourcesTest.scala
@@ -1,0 +1,25 @@
+/*
+ * Copyright and related rights waived via CC0
+ *
+ * You should have received a copy of the CC0 legalcode along with this
+ * work. If not, see <http://creativecommons.org/publicdomain/zero/1.0/>.
+ */
+package org_scalactic.scalactic_3
+
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Assertions.assertThrows
+import org.junit.jupiter.api.Test
+import org.scalactic.Requirements
+
+class ResourcesTest {
+  @Test
+  def failedRequirementLoadsLocalizedFailureMessage(): Unit = {
+    val thrown: IllegalArgumentException = assertThrows(
+      classOf[IllegalArgumentException],
+      () => Requirements.require(false)
+    )
+
+    assertThat(thrown.getMessage).isNotBlank()
+    assertThat(thrown.getMessage).contains("false")
+  }
+}

--- a/tests/src/org.scalactic/scalactic_3/3.3.0-alpha.1/user-code-filter.json
+++ b/tests/src/org.scalactic/scalactic_3/3.3.0-alpha.1/user-code-filter.json
@@ -1,0 +1,13 @@
+{
+  "rules" : [
+    {
+      "excludeClasses" : "**"
+    },
+    {
+      "includeClasses" : "org.scalactic.**"
+    },
+    {
+      "includeClasses" : "org_scalactic.scalactic_3.**"
+    }
+  ]
+}

